### PR TITLE
Releasing v3.2.4

### PR DESCRIPTION
**Pull request by azizk**
:grinning:

### CODE REVIEW

File: commands/mix_test.py [hunk 1]
```diff
@@ -471,10 +478,20 @@ def call_mix_test_with_settings(window, **params):
...
+  global IS_MIX_TEST_RUNNING
+
+  if IS_MIX_TEST_RUNNING:
+    sublime.message_dialog('The `mix test` process is still running!')
+    return
+
+  IS_MIX_TEST_RUNNING = True
+
+  cmd, file_path, names, lines, seed, failed, args = \
+    list(map(mix_params.get, ('cmd', 'file_path', 'names', 'lines', 'seed', 'failed', 'args')))
```
Comment (azizk) [inline commands/mix_test.py:494]: Seems buggy. Pls don't merge yet.

**Comment by princemaple** [comment]
let me know when it's ready 😉

**Comment by princemaple** [comment]
or feel free to release it yourself :)

**Comment by princemaple** [comment]
ready to release?

**Comment by azizk** [comment]
Not yet, not yet. :sweat_smile: I haven't found time to focus on finishing this up. Hope it will be soon.

**Comment by princemaple** [comment]
No worries. I saw that the version number was bumped and thought it might be ready now 😅

Take your time and don't burn out!

**Comment by azizk** [comment]
I renamed the branch and so I'm closing this. Will open a new PR. :+1:

**Pull request closed without merging**